scripts/deployToDB.py

```diff
@@ -167,6 +183,74 @@ def update_config(data_diff, selector):
 
 def update_diff_db(selector, item_name=None):
     final_report = []
+    # check if selector is valid
```

Comment:
It's better to write separate .py script for accounts CI/CD. And this script has to be dependency for sources/destinations CI/CD flow as account names will be referred in destination definitions

Reply:
common functions can still be re-used from this script